lib/rosette/tms/smartling-tms/smartling_file.rb

```diff
@@ -133,6 +145,20 @@ def serializer_const
           @serializer_const ||= Rosette::Core::SerializerId.resolve(serializer_id)
         end
 
+        def commit_log
+          @commit_log ||= rosette_config.datastore.lookup_commit_log(
+            repo_config.name, commit_id
+          )
+        end
+
+        def granularity
+          configurator.phrase_storage_granularity
+        end
+
+        def rev_commit
+          @rev_commit ||= repo_config.repo.get_rev_commit(commit_id)
```

Comment:
will this be ok with merging master back into the user's dev branch? I think the commit will be available again (even though it wasn't technically theirs) on their branch instead of master.